BHoM/BHoM_Engine
# Structure_Engine: Description query

### Issues addressed by this PR
A general description to profiles, sections and surface properties.
WIP adding a NameOrDescription method that outputs the description name if there is no name.

Closes #820 

### Test files
https://burohappold.sharepoint.com/sites/BHoM/02_Current/Forms/AllItems.aspx?csf=1&e=CJWjNz&cid=0b727f4a%2Daf39%2D4d1d%2D8942%2D0c26c33d7c20&FolderCTID=0x0120008122C8891F89054B8ACED0196C70DFC4&viewid=a5a93cba%2Dfcca%2D46cc%2Da31a%2D80b8d5cbfd7b&id=%2Fsites%2FBHoM%2F02%5FCurrent%2F12%5FScripts%2F01%5FTest%20Scripts%2FBHoM%5FEngine%2FStructure%5FEngine%2FStructure%5FEnigne%2DPR1295%2D%20DescriptionQuery

How to test:
Description: Look at the pannels are see if the names are appropriate.
DescriptionOrName: Look at the pannels and see if the component only renames into description if there isn't already a name.
